lib/Controller/ApiController.php

```diff
@@ -83,6 +90,12 @@ public function __construct(
 	 * @throws OCSException
 	 */
 	public function getRelatedResources(string $providerId, string $itemId): DataResponse {
+		if (is_null($this->circlesManager)) {
+			$this->logger->log(2, 'RelatedResources require Circles');
```

Comment:
```suggestion
			$this->logger->info('RelatedResources require Circles');
```
No magic number please. (Not sure if info is the right level)